packages/kbn-shared-ux-components/src/toolbar/buttons/add_from_library/add_from_library.tsx

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { ToolbarButton, Props as ToolbarButtonProps } from '../primary/primary';
+
+export type Props = Omit<ToolbarButtonProps, 'iconType' | 'label'>;
+
+const label = {
+  getLibraryButtonLabel: () =>
+    i18n.translate('sharedUXComponents.toolbar.buttons.addFromLibrary.libraryButtonLabel', {
+      defaultMessage: 'Add from library',
+    }),
+};
+
+export const AddFromLibraryButton = ({ onClick, ...rest }: Props) => (
+  <ToolbarButton
+    {...rest}
+    iconType="folderOpen"
+    onClick={onClick}
```

Comment:
This can be with `rest`, since we're not actually using it for anything

Reply:
[63d8c31](https://github.com/elastic/kibana/pull/127605/commits/63d8c31f0de1ff81e9cff47f29178a03f01fc9d7) for changes thank you!